app/templates/profile.hbs

```diff
@@ -1,13 +1,48 @@
 <div class="container">
-<br>
-<br>
-<div class="row">
-  <div class="col-md-12">
-    <div class="input-group">
-      <span class="input-group-addon">OAuth2 Token</span>
-      <input type="text" class="form-control" value={{session.data.authenticated.user.token}} readonly>
+    <br>
+    <br>
+    <div class="row">
+        <div class="col-xs-12">
+            <h1> My Profile</h1>
+        </div>
     </div>
-  </div>
-</div>
-{{outlet}}
+    <br>
+    <div class="row">
+        <div class="col-md-12">
+            <div class="input-group">
+                <span class="input-group-addon">OAuth2 Token</span>
+                <input type="text" class="form-control" value={{session.data.authenticated.user.token}} readonly>
+            </div>
+        </div>
+    </div>
+    {{#if model}}
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="row">
+                    <div class="col-xs-12">
+                        <h2>Your provider registration requests</h2>
+                    </div>
+                </div>
+                {{#each model as |registration|}}
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <strong>Status: </strong><span class="{{status registration.status}}">{{registration.status}}</span>
```

Comment:
Could we add in a some popover, maybe with a question mark icon, to give the user information about what each status means?

`pending`: "The application for `source name` is being reviewed. Check back soon!"
`accepted`: "`source name` have been approved as a source. Start pushing data!(hyperlink to pushapi docs or placeholder)" / "`source name` have been approved as a source but we have not started scraping it yet"
`implemented`: "`source name` is now a source. Check it out! (hyperlink to discover filtered by this source)"
`rejected`: "`source name` has been rejected as a source. Contact support@osf.io for additional information"

ping @rickjohnson for language opinions